src/utilities/utility.ts

```diff
@@ -129,6 +131,93 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
     return commandExecutionResults;
 }
 
+export async function getFilePathsConfigs(): Promise<any> {
+
+    let filePathsConfig: any = {};
+    const MANIFEST_PATHS_KEY = 'manifestFilePaths';
+    const HELM_CHART_KEY = 'helmChartFilePaths';
+    const DOCKERFILE_PATH_KEY = 'dockerfilePaths';
+    const DOCKERFILE_PATH_LABEL_KEY = 'dockerfile-path';
+
+    let inputManifestFiles = inputParams.manifests || [];
+    filePathsConfig[MANIFEST_PATHS_KEY] = inputManifestFiles;
+
+    let helmChartPaths = (process.env.HELM_CHART_PATHS && process.env.HELM_CHART_PATHS.split('\n').filter(path => path != "")) || [];
+    filePathsConfig[HELM_CHART_KEY] = helmChartPaths;
+
+    //Parsing dockerfile paths for images
+    let imageNames = core.getInput('images').split('\n');
+    let imageDockerfilePathMap: any = {};
+    let pathKey: any, pathVal: any;
+
+    //Fetching from env var if available
+    let dockerfilePathsList: any[] = (process.env.DOCKERFILE_PATHS &&  process.env.DOCKERFILE_PATHS.split('\n')) || [];
+    dockerfilePathsList.forEach(path => {
+        if(path){
+            pathKey = path.split(' ')[0];
+            pathVal = path.split(' ')[1];
+            imageDockerfilePathMap[pathKey] = pathVal;
+        }
+    })
+
+    //Fetching from image lable if available
+    for(const image of imageNames){
+        let args: string[] = [image];
+        let resultObj: any;
+        let containerRegistryName = image;
+
+        try{
+            let usrname = process.env.CR_USERNAME || null;
+            let pwd = process.env.CR_PASSWORD || null;

```

Comment:
`password`

Reply:
You could also change this to registryUsername and registryPassword

Reply:
Done